api/model/storage/postgres/timeseries.go

```diff
@@ -308,35 +330,32 @@ func (s *Storage) FetchTimeseries(dataset string, storageName string, timeseries
 			return nil, err
 		}
 	}
-
-	// sort the timeseries
-	sort.Slice(response, func(i, j int) bool {
-		return response[i].Time < response[j].Time
-	})
-
-	// sum duplicate timestamps
-	response = removeDuplicates(response)
```

Comment:
Are the sort and duplicate summing still happening elsewhere?  This is needed for cases where you series key settings don't yield a single timeseries.  Its hardcoded to a sum right now, but should support different aggregations down the road.